pybossa/api/task.py

```diff
@@ -54,10 +54,14 @@ def _forbidden_attributes(self, data):
                 raise BadRequest("Reserved keys in payload")
 
     def _update_attribute(self, new, old):
-        if (new.state == 'completed') and (old.n_answers <= new.n_answers):
-            new.state = 'ongoing'
-        if not new.gold_answers:
-            new.calibration = 0
+        gold_task = bool(new.gold_answers)
+        if (new.state == 'completed'):
+            n_task_taskruns = task_repo.get_task_run_by(id=new.id)
```

Comment:
you should be able to use `task.task_runs` for this, see https://github.com/bloomberg/pybossa/blob/master/pybossa/model/task.py#L65. Also can we just call it `n_taskruns`?